apps/desktop-wallet/src/hooks/useThrottledGitHubApi.ts

```diff
@@ -18,22 +18,27 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 
 import { useState } from 'react'
 
-import { AppMetaData, isRcVersion, KEY_APPMETADATA, toAppMetaData } from '@/utils/app-data'
+import { AppMetaData, AppMetadataGitHub, isRcVersion, KEY_APPMETADATA, toAppMetaData } from '@/utils/app-data'
 import { useTimeout } from '@/utils/hooks'
 
 // TODO: Move to shared
 const ONE_HOUR_IN_MS = 1000 * 60 * 60
 
-const useThrottledGitHubApi = (githubApiCallback: (appMetadata: AppMetaData) => Promise<void>) => {
+interface ThrottledGitHubApiProps {
+  key: keyof AppMetadataGitHub
+  githubApiCallback: (appMetadata: AppMetaData) => Promise<void>
+}
+
+const useThrottledGitHubApi = ({ key, githubApiCallback }: ThrottledGitHubApiProps) => {
```

Comment:
Nit: what about renaming `key` to a more specific: `lastGithubCallTimestampKey`

Reply:
Fixed